website/src/components/Schema/index.js

```diff
@@ -148,13 +161,15 @@ const Schema = ({ schema, menuItem, isLatestSchema }) => {
       Header: 'Attributes',
       id: 'attributes',
       Cell: ({ original: { restrictions, meta } }) => {
+        const tags = [];
         if (restrictions && restrictions.required) {
-          return <Tag type={TAG_TYPES.required} />;
-        } else if (meta && !!meta.dependsOn) {
-          return <Tag type={TAG_TYPES.dependency} />;
-        } else {
-          return null;
+          tags.push(<Tag type={TAG_TYPES.required} />);
+        }
+
+        if (meta && !!meta.dependsOn) {
+          tags.push(<Tag type={TAG_TYPES.dependency} />);
         }
+        return tags.length > 0 ? <TagContainer>{tags}</TagContainer> : null;
```

Comment:
what breaks if `TagContainer` is shown without tags? This seems much simpler if written as:
```
<TagContainer>
  {restrictions && restrictions.required && <Tag type={TAG_TYPES.required} />}
  {meta && !!meta.dependsOn && <Tag type={TAG_TYPES.dependency} />}
</TagContainer>
```

Reply:
more readable with:
```
const isRestrictedField = restrictions && restrictions.required
const isDependentField = meta && !!meta.dependsOn

<TagContainer>
  {isRestrictedField && <Tag type={TAG_TYPES.required} />}
  {isDependentField && <Tag type={TAG_TYPES.dependency} />}
</TagContainer>
```

Reply:
way more readable thanks @hlminh2000